civic-tech-club/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Step 1: Base Stage for both Dev and Prod
+FROM node:16 AS base
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package.json and install dependencies
+COPY package*.json ./
+RUN npm install
+
+# Install Sass globally
+RUN npm install -g sass
```

Comment:
If we go the recommended vite route, then this won't be necessary. See also my comment below about adding all dependencies to package.json.

Reply:
vite is implemented